nixos/modules/services/networking/networkmanager.nix

```diff
@@ -173,13 +173,24 @@ in {
         '';
       };
 
+      withSupplicant = mkOption {
+        internal = true;
```

Comment:
Not sure about the internal here.  Also we could use a more generic option like "ignoreWireless"/ "delegateWireless" without naming supplicant as there might be other options in the future.

Reply:
I'm not exactly sure about the semantics of `internal`.
Actually I think I want it to be just a variable.
Is internal like a config option that the user cannot set themself, just the module?
Yeah okay, not calling it supplicant makes sense.

Reply:
I switched to a variable with the name `delegateWireless`.